torchtext/data/field.py

```diff
@@ -143,7 +143,7 @@ class Field(RawField):
     def __init__(self, sequential=True, use_vocab=True, init_token=None,
                  eos_token=None, fix_length=None, dtype=torch.long,
                  preprocessing=None, postprocessing=None, lower=False,
-                 tokenize=None, tokenizer_language='en', include_lengths=False,
```

Comment:
I'd definitely consider this a BC-breaking change, especially since 'en' is still the correct choice for tokenizers other than spacy.